pkg/ingester/user_metrics_metadata_test.go

```diff
@@ -107,15 +108,132 @@ func TestUserMetricsMetadata(t *testing.T) {
 			}
 
 			// Verify expected elements are stored
-			clientMeta := mm.toClientMetadata()
+			clientMeta := mm.toClientMetadata(nil)
 			assert.ElementsMatch(t, testData.expectedMetadata, clientMeta)
 
 			// Purge all metadata
 			mm.purge(time.Time{})
 
 			// Verify all metadata purged
-			clientMeta = mm.toClientMetadata()
+			clientMeta = mm.toClientMetadata(nil)
 			assert.Empty(t, clientMeta)
 		})
 	}
 }
+
+type dummytest struct{}
```

Comment:
```suggestion
type dummyTest struct{}
```

I'd also add a comment to explain what is its purpose. Or maybe rename to `noopTestingT`?